build.sbt

```
@@ -93,9 +95,10 @@ lazy val tyrian =
     .jsSettings(
       commonJsSettings,
       libraryDependencies ++= Seq(
-        "org.scala-js"  %%% "scalajs-dom"        % Dependancies.scalajsDomVersion,
-        "org.typelevel" %%% "cats-effect-kernel" % Dependancies.catsEffect,
-        "co.fs2"        %%% "fs2-core"           % Dependancies.fs2
+        "org.scala-js"     %%% "scalajs-dom"        % Dependancies.scalajsDomVersion,
+        "org.typelevel"    %%% "cats-effect-kernel" % Dependancies.catsEffect,
+        "co.fs2"           %%% "fs2-core"           % Dependancies.fs2,
+        "io.github.buntec" %%% "scala-js-snabbdom"  % "0.0-1c017a3-SNAPSHOT"
```

```suggestion
        "io.github.buntec" %%% "scala-js-snabbdom"  % "0.1.0-M2"
```

https://repo1.maven.org/maven2/io/github/buntec/scala-js-snabbdom_sjs1_3/0.1.0-M2/